### PR TITLE
add Selenium test1

### PR DESCRIPTION
**Pull request by Roman-Derback**

### CODE REVIEW

File: Projects/Team2/derbackAPP/SeleniumTest.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            driver.FindElement(By.CssSelector("#loginForm > div.column > app-email-input > div > input")).Click();
+            driver.FindElement(By.CssSelector("#loginForm > div.column > app-email-input > div > input")).SendKeys("derback2015@ukr.net");
+            Thread.Sleep(1000); 
+            driver.FindElement(By.CssSelector("#loginForm > div.column > app-password-input > div > input")).Click();
+            driver.FindElement(By.CssSelector("#loginForm > div.column > app-password-input > div > input")).SendKeys("GoodParol100%");
+            Thread.Sleep(1000);
+            driver.FindElement(By.XPath("/html/body/div[1]/div/div/div/div/form/div[2]/app-form-button/div/button")).Click();
+        }
+
+        static void Main(string[] args)
```
Comment (romanukhaav) [inline Projects/Team2/derbackAPP/SeleniumTest.cs]: you can remove this empty method
https://stackoverflow.com/questions/17095217/does-not-contain-a-static-main-method-suitable-for-an-entry-point

File: Projects/Team2/derbackAPP/SeleniumTest.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            driver.Quit();
+        }
+        [Test]
+        public void TestPubg()
+        {
+            string link = "https://lite.pubg.com/ru/news/";
+            driver.Navigate().GoToUrl("https://lite.pubg.com/ru/download/");
+            driver.Navigate().GoToUrl("https://lite.pubg.com/ru/news/");
+            Assert.AreEqual(link, driver.Url);
+            Thread.Sleep(2000);
```
Comment (romanukhaav) [inline Projects/Team2/derbackAPP/SeleniumTest.cs]: try to avoid Thread.Sleep
it is better just to increase ImplicitWait

**Review by romanukhaav — DISMISSED** [review]

File: Projects/Team2/derbackAPP/SeleniumTest.cs (hunk 2 above)
Comment (Roman-Derback) [reply Projects/Team2/derbackAPP/SeleniumTest.cs]: many threadsleep through my bad internet

File: Projects/Team2/derbackAPP/SeleniumTest.cs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            driver.Navigate().GoToUrl("https://lite.pubg.com/ru/download/");
+            driver.Navigate().GoToUrl("https://lite.pubg.com/ru/news/");
+            Assert.AreEqual(link, driver.Url);
+            driver.FindElement(By.XPath("/html/body/div[2]/header/div/div[2]/aside/ul/li[1]/a")).Click();
+            Thread.Sleep(1000);
+            driver.FindElement(By.CssSelector("#loginForm > div.column > app-email-input > div > input")).Click();
+            driver.FindElement(By.CssSelector("#loginForm > div.column > app-email-input > div > input")).SendKeys("derback2015@ukr.net");
+            driver.FindElement(By.CssSelector("#loginForm > div.column > app-password-input > div > input")).Click();
+            driver.FindElement(By.CssSelector("#loginForm > div.column > app-password-input > div > input")).SendKeys("GoodParol100%");
+            driver.FindElement(By.XPath("/html/body/div[1]/div/div/div/div/form/div[2]/app-form-button/div/button")).Click();
```
Comment (VladislavBr) [inline Projects/Team2/derbackAPP/SeleniumTest.cs]: you go to the news site and check if you are there. but you need to check if the registration was successful

**Review by VladislavBr — COMMENTED** [review]
well well well

File: Projects/Team2/derbackAPP/SeleniumTest.cs [hunk 4]
```diff
@@ -32,9 +32,21 @@ public void TestPubg()
...
+            //
             driver.FindElement(By.CssSelector("#loginForm > div.column > app-password-input > div > input")).Click();
             driver.FindElement(By.CssSelector("#loginForm > div.column > app-password-input > div > input")).SendKeys("GoodParol100%");
+            //check password
+            var parolText = driver.FindElement(By.CssSelector("#loginForm > div.column > app-password-input > div > input")).GetAttribute("value");
+            Assert.AreEqual(parolText, "GoodParol100%");
+            //
             driver.FindElement(By.XPath("/html/body/div[1]/div/div/div/div/form/div[2]/app-form-button/div/button")).Click();
+            //check how change Button (login/logout) 
+            var buttonlog = driver.FindElement(By.CssSelector("#logoutComponent > a > span")).Text;
```
Comment (Roman-Derback) [inline Projects/Team2/derbackAPP/SeleniumTest.cs:47]: Thread.Sleep(1000); forgot))

**Review by VladislavBr — COMMENTED** [review]
generally not bad. Elon musk respects

**Review by romanukhaav — APPROVED** [review]

**Pull request merged**